README.md

```diff
@@ -70,15 +70,22 @@ CI runs a series of steps;  this the sequence to do it locally, along with some
     bin/rake
     ```
```

Comment:
Note: The diffs view makes it hard to see what changed, since some stuff just got moved around.  May be easier to just view the whole file to see what it looks like now.